### PR TITLE
Add mentor bio for ansumanbebarta

### PR DESCRIPTION
**Pull request by ansumanbebarta**

### CODE REVIEW

File: mentors/python/a.json [hunk 1]
```diff
@@ -103,4 +103,12 @@
     "avatar_url": null,
     "bio": "Web software developer and data geek. I usually code in Python and JavaScript, sometimes in ruby. Always learning and happy to help others in their programming development "
   }
```
Comment (rpottsoh) [inline mentors/python/a.json:105]:
```suggestion
  },
```
Missing a comma.

**Review by rpottsoh — CHANGES_REQUESTED** [review]

**Comment by rpottsoh** [comment]
@ansumanbebarta your second commit fixed the issue.  Your third commit has added your bio a second time.

**Comment by ansumanbebarta** [comment]
This was my first PR ever, so I was unaware of the process of updating a PR.  I am sorry for the inconvenience.

**Comment by hugovk** [comment]
I don't think you are a mentor for this project.

Please don't create spam PRs for Hacktoberfest, it's against the rules and values, and not fair on maintainers.

https://hacktoberfest.digitalocean.com/details#spam

Maintainers: please label this `invalid` so it's not counted towards Hacktoberfest.

**Comment by neenjaw** [comment]
Hey @hugovk, there really isn't a good way to externally see if someone is or isn't an active mentor on Exercism as many good mentors aren't overly active on github or in the exercism slack chat, so they operate in relative obscurity. @ihid is probably best to sort this out as he is the founder and can do some investigation/asking around.

I can see where you are coming from though, there has been a flurry across the whole of github to make low-quality PRs, but I am not sure this one is so straightforward.

**Review by SleeplessByte — APPROVED** [review]

**Comment by SleeplessByte** [comment]
Welcome to Python 🐍

**Comment by iHiD** [comment]
@ansumanbebarta This wasn't a fun process for you. I'm sorry. Thank you for being a mentor. You should receive an invitation to Slack too. Please say hello when you get there :) I'm @iHiD 1-1.

@hugovk Thank you for trying to help, but please leave the Hacktoberfest "policing" to the senior maintainers team.

**Comment by hugovk** [comment]
I'd like to sincerely apologise, I was off mark.

I wish you all the best and hope this doesn't put you off!

**Comment by iHiD** [comment]
Thanks @hugovk 💙

**Pull request merged**